CONTRIBUTING.md

```diff
@@ -6,6 +6,8 @@ Contributions to gittuf can be of several types:
   enhancements to the implementation
 * new issues or feature requests
 
+You can also join our community in #gittuf in [OpenSSF Slack](https://openssf.slack.com/) and/or our monthly gittuf Community Meetings, which take place the first Friday of the month at 17:00 UTC ([your timezone](https://time.is/1700_UTC)). Here are the prior [meeting notes](https://docs.google.com/document/d/1tXFCVUHsICLpLKxcGvhzBDUWmpsY1LQvysFaX6AJRkk/edit#heading=h.9m0zi4b0wnne) to get oriented.
```

Comment:
What do you think about pointing to https://github.com/gittuf/community/blob/main/README.md#join-us so we have this information in a single place?

Reply:
nit: Also, could you please wrap the final text to 80 cols? :)

Reply:
That is indeed a _much_ better plan (sorry, I missed that!), so yes, will do! And no worries on 80 char wrap, I wasn't sure. :D

Reply:
Thanks! Also, could you DCO sign off to make the bot happy?